### DISCUSSION TITLE
Jump 시 애니메이션

### DISCUSSION THREAD

**Discussion by peterkangdev** [naverz/zepeto-studio-kor#332] [5. Q&A]
JumpPower = 10 
이렇게 주고 점프하면, 
점프 후 바닥에 닿지도 않았는데, 점프 idle 애니메이션이 끝나더라구요. 
그래서 공중에서 멀뚱이 서 있습니다.
아무래도 코드를 수정해야 할것 같은데, 아시다시피 수정이 않되고요.
어떻게 하면 좋을까요?

**Comment by x4cf** [comment]
점프 하강시, 제스처를 실행시키시는게 좋은 방법이 될거같아요 :)
